web/themes/custom/subtheme_videogame/src/components/views/_view--catalog.scss

```diff
@@ -14,6 +14,10 @@
     margin-bottom: 2rem;
   }
 
+  &__catimage {
```

Comment:
If you want to create a variant you should foloww the BEM standards, e.g. `.catalog__image--somevariant`.

Reply:
I renamed it to .catalog__image--marginless.